.github/workflows/lint.yml

```diff
@@ -50,20 +49,29 @@ jobs:
       - name: Check
         working-directory: build-lint
         run: |
+          # Using a working directory for cppcheck makes analysis faster
+          mkdir cppcheck-wd
           # Sanitize AMREX_GIT_VERSION define with 3 backslashes and quotes which cppcheck can't parse
           sed -i '' -e 's/\\\\\\"//g' compile_commands.json
+          # Cppcheck ignores -isystem directories, so we change them to regular -I include directories (with no spaces either)
+          sed -i '' -e 's/isystem /I/g' compile_commands.json
```

Comment:
@jrood-nrel With the latest AMReX dev snapshot, we no longer need to fix the AMReX_GIT_VERSION as it is not longer a compile definition on the command line. Can you remove that?

```
sed -i '' -e 's/\\\\\\"//g' compile_commands.json
```

Reply:
Awesome. I would love to.